lms/djangoapps/ora_staff_grader/mock/data/submissions.json

```diff
@@ -0,0 +1,73 @@
+{
+    "block-a": {
+        "SUBMISSION_ID-0": {
+            "submissionId": "SUBMISSION_ID-0",
+            "username": "USERNAME-0",
+            "dateSubmitted": 1631215154955,
+            "score": null,
+            "gradeData": null,
+            "gradeStatus": "ungraded",
+            "lockStatus": "unlocked"
+        },
+        "SUBMISSION_ID-1": {
+            "submissionId": "SUBMISSION_ID-1",
+            "username": "USERNAME-1",
+            "dateSubmitted": 1631301554955,
+            "score": {
+                "pointsEarned": 70,
+                "pointsPossible": 100
+            },
+            "gradeData": {
```

Comment:
I don't see gradeData in this data object in the doc

Reply:
https://openedx.atlassian.net/wiki/spaces/PT/pages/3154542730/Enhanced+Staff+Grader+Data+Flow+Design#return-data-(%7B-submission-%7D)

Reply:
Yes, but this also appears to be used in the submissionList endpoint, which doesn't include that data

Reply:
Updated